CMakeLists.txt

```diff
@@ -61,3 +62,6 @@ target_sources(vhost-server PRIVATE
   virtio/virtio_blk.c
   virtio/virtio_fs.c
 )
+
+
+add_subdirectory(tests)
```

Comment:
Again, I'm not sure NBS needs the tests subdirectory for their integration (and this might break their isolated build as well)

Reply:
It is very unusual to be selective about build systems and focus on only one consumer for an open source project.

Reply:
Building tests makes sense and NBS will fix their build if building tests break it. So I think this is a good idea to build tests.